packages/vast-xml2js/src/__tests__/index.spec.js

```diff
@@ -0,0 +1,201 @@
+import {
+  get,
+  getAll,
+  getFirstChild,
+  getText,
+  getAttributes,
+  getAttribute,
+  parseXml
+} from '../index';
+
+const xml = `<?xml version="1.0" encoding="utf-8"?>
+<note importance="high" logged="true">
+    <title><![CDATA[Happy]]></title>
+    <todo>Work</todo>
+    <todo>Play</todo>
+</note>`;
+
+describe('parseXml', () => {
+  test('must parse a given xml into a js object', () => {
+    const data = parseXml(xml);
+
+    expect(data).toEqual({
+      elements: [{
```

Comment:
even if you can't do a proper TDD, but in these cases I prefer using jest snapshots

Reply:
I don't